src/main/webapp/app/exam/manage/exams/exam-update.component.html

```diff
@@ -139,11 +128,65 @@ <h4>{{ 'artemisApp.examManagement.editExam' | artemisTranslate }}</h4>
+                <div class="row mb-3" *ngIf="!exam.testExam">
+                    <div class="col-sm-6">
+                        <jhi-date-time-picker
+                            labelName="{{ 'artemisApp.examManagement.publishResultsDate' | artemisTranslate }}"
+                            [(ngModel)]="exam.publishResultsDate"
+                            [error]="!isValidPublishResultsDate"
+                            name="publishResultsDate"
+                            id="publishResultsDate"
+                        ></jhi-date-time-picker>
+                    </div>
+                    <div class="col-sm-6">
+                        <label class="form-check-label" for="numberOfCorrectionRoundsInExam" jhiTranslate="artemisApp.examManagement.numberOfCorrectionRoundsInExam">
+                            Number of correction rounds in Exam
+                        </label>
+                        <input
+                            id="numberOfCorrectionRoundsInExam"
+                            name="numberOfCorrectionRoundsInExam"
+                            class="form-control"
+                            type="number"
+                            min="1"
+                            max="2"
+                            [(ngModel)]="exam.numberOfCorrectionRoundsInExam"
+                        />
+                    </div>
+                </div>
+                <div class="row mb-3">
+                    <div class="col-sm-6">
+                        <jhi-date-time-picker
+                            labelName="{{ 'artemisApp.examManagement.examStudentReviewStart' | artemisTranslate }}"
+                            [(ngModel)]="exam.examStudentReviewStart"
+                            [error]="!isValidExamStudentReviewStart"
+                            name="examStudentReviewStart"
+                            id="examStudentReviewStart"
+                        ></jhi-date-time-picker>
+                    </div>
+                    <div class="col-sm-6">
+                        <jhi-date-time-picker
+                            labelName="{{ 'artemisApp.examManagement.examStudentReviewEnd' | artemisTranslate }}"
+                            [(ngModel)]="exam.examStudentReviewEnd"
+                            [error]="!isValidExamStudentReviewEnd"
+                            name="examStudentReviewEnd"
+                            id="examStudentReviewEnd"
+                        ></jhi-date-time-picker>
+                        <span *ngIf="!isValidExamStudentReviewEnd" class="invalid-feedback"> {{ 'artemisApp.examManagement.reviewDatesInvalidWarning' | artemisTranslate }}</span>
+                    </div>
+                </div>
             </div>
+            <hr />
+            <!-- Section Texts-->
+            <h5 style="padding-bottom: 5px">{{ 'artemisApp.examManagement.sections.text' | artemisTranslate }}</h5>
```

Comment:
Given the amount of occurrences of this styling, defining a dedicated CSS class might indeed be worth considering.